packages/vue-component-library/src/stories/BlockStaffArticleList.stories.js

```diff
@@ -228,6 +228,54 @@ export function FtvaDateRange() {
   }
 }
 
+export function FtvaCategory() {
+  return {
+
+    data() {
+      return { ...mockDateRange2 }
+    },
+    methods: {
+      parseDate(sectionHandle, startDate, endDate, ongoing) {
+        console.log(sectionHandle, startDate, endDate, ongoing)
+
+        if (ongoing)
+          return 'Ongoing'
+        if (sectionHandle === 'ftvaEvent')
+          return formatDates(startDate, startDate, 'shortWithYear')
+        if (sectionHandle === 'ftvaEventSeries')
+          return formatSeriesDates(startDate, endDate, 'shortWithYear')
+
+        return null
+      }
+    },
+    provide() {
+      return {
+        theme: computed(() => 'ftva'),
+      }
+    },
+    components: { BlockStaffArticleList },
+    template: `
+      <block-staff-article-list
+          :image="image"
+          :to="to"
+          category=ULLAMCO
```

Comment:
@angelahuqing Output looks good. For code style consistency, add the category data to the mock object; then call it in the component with the v-bind directive (`:category`).